codalab/lib/bundle_cli.py

```diff
@@ -812,6 +812,31 @@ def do_search_command(self, argv, parser):
                 client.add_worksheet_item(worksheet_uuid, worksheet_util.bundle_item(bundle_uuid))
             worksheet_info = client.get_worksheet_info(worksheet_uuid, False)
             print 'Added %d bundles to %s' % (len(bundle_uuids), self.worksheet_str(worksheet_info))
+        return self.create_structured_info_map([('refs', reference_map),])
+
+    def create_structured_info_map(self, structured_info_list):
+        '''
+        Return dict of info dicts (eg. bundle/worksheet reference_map) containing
+        information associated to bundles/worksheets. cl wls, ls, etc. show uuids
+        which are too short. This dict contains additional information that is
+        needed to recover URL on the client side.
+        '''
+        return { k: v for k, v in structured_info_list}
```

Comment:
You can also do the same with the dict constructor without the comprehension: `dict(structured_info_list)`